internal/world/world.go

```diff
@@ -81,6 +82,11 @@ func (w *World) Funcs() template.FuncMap {
 	funcs["vault"] = func(path, field string) string {
 		return w.Vault().Secret(path, field)
 	}
+
+	for name, fn := range sprig.FuncMap() {
```

Comment:
Thank you :) Could you please change the order here so that sprig functions can be overwritten by functions inside the original `funcs`?